osu.Game/Graphics/UserInterface/HoverSampleDebounceComponent.cs

```diff
@@ -5,19 +5,22 @@
 
 using osu.Framework.Allocation;
 using osu.Framework.Bindables;
-using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics;
 using osu.Framework.Input.Events;
 using osu.Game.Configuration;
+using osuTK;
 
 namespace osu.Game.Graphics.UserInterface
 {
     /// <summary>
     /// Handles debouncing hover sounds at a global level to ensure the effects are not overwhelming.
     /// </summary>
-    public abstract partial class HoverSampleDebounceComponent : CompositeDrawable
+    public abstract partial class HoverSampleDebounceComponent : Component
     {
         private Bindable<double?> lastPlaybackTime;
 
+        public override bool ReceivePositionalInputAt(Vector2 screenSpacePos) => Parent?.ReceivePositionalInputAt(screenSpacePos) ?? false;
```

Comment:
I think

```suggestion
        public override bool ReceivePositionalInputAt(Vector2 screenSpacePos) => Parent?.ReceivePositionalInputAt(screenSpacePos) == true;
```

was the preferred style. Also weird that the base call is not checked at all (although I guess it must always return `true`, because the class has `OnHover()` overridden).